library/java/src/io/envoyproxy/envoymobile/engine/JvmFilterContext.java

```diff
@@ -117,7 +117,13 @@ public Object onResponseTrailers(long trailerCount) {
   }
 
   /**
+   * Invokes onResumeRequest callback with pending HTTP entities.
    *
+   * @param headerCount,  total pending headers included in the header block.
+   * @param data,         buffered body data.
+   * @param trailerCount, total pending trailers included in the trailer block.
+   * @param endStream,    whether the stream is closed at this point.
+   * @return Object[],    tuple of status with updated entities to be forwarded.
```

Comment:
Isn't this only an `Object`?

Reply:
technically it is an `Object[]` but it's declared as `Object` to make the JNI lookup consistent.